src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -723,11 +749,11 @@ def _create_pgm_input_transformers(self):  # pylint: disable=too-many-statements
         pgm_transformers["clock"] = clocks
         pgm_transformers["winding_from"] = winding_types["winding_from"]
         pgm_transformers["winding_to"] = winding_types["winding_to"]
-        pgm_transformers["tap_nom"] = tap_nom
-        pgm_transformers["tap_pos"] = tap_pos
+        pgm_transformers["tap_nom"] = tap_nom.astype("i4")  # TODO(mgovers) shouldn't this be rounded?
+        pgm_transformers["tap_pos"] = tap_pos.astype("i4")  # TODO(mgovers) shouldn't this be rounded?
```

Comment:
@nitbharambe  can you answer this?

Reply:
There isnt a need to be rounded, since the value would be casted to the pgm's numpy type anyway.